lib/spy.js

```diff
@@ -521,6 +592,16 @@ module.exports = function (chai, _) {
         , 'expected ' + this._obj + ' to have not always been called with exactly #{exp}'
         , args
       );
+    } else if(_with && nthCall) {
+      var ordinalNumber = generateOrdinalNumber(nthCall),
+          actArgs = calls[nthCall - 1];
+      new Assertion(this._obj).to.be.have.been.called.min(nthCall);
+      this.assert(
+          _.eql(actArgs, args)
+        , 'expected ' + this._obj + ' to have been called at the ' + ordinalNumber + ' time with exactly #{exp} but got ' + args
```

Comment:
same here I guess `args` can be an array of objects and if so, then you will have not well formatted error messages